README.md

```diff
@@ -134,3 +134,11 @@ metadata: {}
 resourceVersion: ""
 selfLink: ""
 ```
+
+## Building Type Changes
+If any changes are made to the alarm definition type under pkg, follow these steps
+
+Setup:
+1) Ensure https://github.com/kubernetes/code-generator is present in the vendor directory. It may need to be manually added.
+ This is required to run the code generation scripts.
+2) Run ```./hack/update-codegen.sh``` from the main repo directory.
```

Comment:
Will this update all of the files you have take from k8s?

Reply:
Yes, it will regenerate all the required k8s code in the pkg directory